docs/build/guides/building-a-voting-dapp.md

```diff
@@ -771,19 +771,24 @@ Next step is to fund your `testnet` account using this [faucet](http://faucet.te
 Here's a snippet that you can use to list all deployed modules by using the top-level `list-modules` built-in function:
 
 ```javascript
-const { PactCommand } = require('@kadena/client');
-const { createExp } = require('@kadena/pactjs');
+const { PactCommand } = require("@kadena/client");
+const { createExp } = require("@kadena/pactjs");
```

Comment:
When changing the quoting it makes for a hard to review PR. In our JS code we use single quotes, so it makes sense to keep it that way. Feel free to suggest and change this convention, but please don't mix that up in a single PR.

Reply:
Yeah, that's a good suggestion! I'm used to just running prettier, will revert this 👍